libs/ui/src/lib/steps/steps/steps.component.html

```diff
@@ -4,61 +4,81 @@ <h1 *ngIf="title" class="cvi-steps__title" dataAttribute="steps_title">{{ title
+                      {{ i + 1 }}
+                    </cvi-ng-circle>
+                    <span class="cvi-steps__list-item-title">{{ stepTitle }}</span>
+                  </cvi-ng-track>
+                </div>
+                <cvi-ng-track [gap]="4">
+                  <cvi-ng-icon [name]="'arrow_a_right'" class="cvi-steps__list-item-arrow-icon-wrapper"
+                               [svgClass]="'cvi-steps__list-item-arrow-icon'"></cvi-ng-icon>
+                </cvi-ng-track>
               </cvi-ng-track>
-            </div>
-            <div class="cvi-steps__list-item-circle-wrapper--mobile">
-              <cvi-ng-track [gap]="4" [verticalAlignment]="'center'">
-                <cvi-ng-circle
-                  size="m"
-                  style="--cvi-circle-border-color: var(--cvi-color-sapphire-blue-10); --cvi-circle-color: var(--cvi-color-sapphire-blue-10);"
-                  [progressPercentage]="anyStepSelected ? currentProgressCSSVar : undefined">
-                  {{ i + 1 }}
-                </cvi-ng-circle>
-                <span class="cvi-steps__list-item-title">{{ stepTitle }}</span>
+            </button>
+          </li>
+          <li *ngIf="urlStepTitle"
+              class="cvi-steps__list-item cvi-steps__list-item--is-url-step">
+            <button class="cvi-steps__list-item-button" style="background-color: white;" (click)="urlStepClicked()">
+              <cvi-ng-track [layout]="'flex'" [horizontalAlignment]="'justify'" [verticalAlignment]="'center'">
+                <div class="cvi-steps__list-item-circle-wrapper--desktop">
+                  <cvi-ng-track [gap]="2" [verticalAlignment]="'center'">
+                    <cvi-ng-circle [iconName]="'arrow_a_right'"
+                                   [theme]="'light'"
+                                   [severity]="'info'"
+                                   style="--cvi-circle-border-color: var(--cvi-color-sapphire-blue-13)">
+                    </cvi-ng-circle>
+                    <span class="cvi-steps__list-item-title">{{ urlStepTitle }}</span>
+                  </cvi-ng-track>
+                </div>
               </cvi-ng-track>
-            </div>
-            <cvi-ng-track [gap]="4">
-              <cvi-ng-icon [name]="'arrow_a_right'" class="cvi-steps__list-item-arrow-icon-wrapper"
-                           [svgClass]="'cvi-steps__list-item-arrow-icon'"></cvi-ng-icon>
-            </cvi-ng-track>
-          </cvi-ng-track>
+            </button>
+          </li>
+        </ol>
+        <div #rightEdge style="width: 1px"></div>
+        <button *ngIf="isScrollable && !leftEdgeVisible"
+                aria-label="left-scroll-button"
+                class="cvi-steps__scroll-button cvi-steps__scroll-button--left-scroll"
+                (click)="onClickScroll(-150)">
+          <cvi-ng-icon [name]="'arrow_a_left'"
+                       svgClass="cvi-steps__scroll-button--scroll-icon">
```

Comment:
The class of an icon inside of a scroll button doesn't conform to BEM guidelines. The current class is a "modifier" to `__scroll-button` "element", and modifiers can only be added on the same node where the "element" class is. 

For a separate DOM node we should employ another "element", eg `__scroll-button-icon`.

Reply:
Thanks for explaining. I added fix.